vtexdocs/devportal
# Feature/landing page

#### What is the purpose of this pull request?

Add the landing page, header and footer components

#### What problem is this solving?

Lack of header and footer componente and landing page

#### How should this be manually tested?

[Documented Here 📖 ](https://www.notion.so/vtexhandbook/Componentiza-o-da-Landing-Page-79220ff2a8124481aadd25d3e19732f0)

Review the code and go to [this page](https://deploy-preview-7--elated-hoover-5c29bf.netlify.app/) and check:

- Animations: All cards in Education Section, Updates Section, and Documentation section have animations. Check if the header hides when scrolling down and shows back up when scrolling up

https://user-images.githubusercontent.com/42784961/159562450-d8651cc6-b1c6-4f6d-ace1-a868eeee5d83.mp4

https://user-images.githubusercontent.com/42784961/159563277-31fc5545-f7b8-4533-831b-efdac831822a.mp4

https://user-images.githubusercontent.com/42784961/159562469-7316ca92-13c5-48cc-b33b-6b9bcc1d65e0.mp4

- Links: The footer links, header icon and feedback, all cards in Education and Documentation sections.

- Set the screen view to 1366x768 and compare it with [that component in Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=2044%3A98311). (The landing page was not prepared to compare with others page-sizes from Figma)

https://user-images.githubusercontent.com/42784961/159562973-8d873559-564d-405b-8338-cdc3bbb40dea.mp4

#### Types of changes
